src/mmrelay/matrix_utils.py

```
@@ -1724,6 +1632,148 @@ async def _resolve_alias(alias: str) -> str | None:
+                return cast(str, room_id)
+            error_details = (
+                getattr(response, "message", response)
+                if response
+                else "No response from server"
+            )
+            logger.warning(f"Could not resolve alias {alias}: {error_details}")
+        except NIO_COMM_EXCEPTIONS:
+            logger.exception(f"Error resolving alias {alias}")
+        except (TypeError, ValueError, AttributeError):
+            logger.exception(f"Error resolving alias {alias}")
+        except Exception:  # noqa: BLE001 — keep bridge alive on unexpected alias errors
```

Ruff (0.14.14)

Unused `noqa` directive (unused: `BLE001`)

Remove unused `noqa` directive

(RUF100)